utils/jwtUtil.js

```diff
@@ -94,8 +94,12 @@ function getUsernameFromToken(req) {
           return reject(err);
         }
 
-        const username = decodedData.username;
-        return resolve(username);
+        const resObj = {
+          username: decodedData.username,
+          tokenname: decodedData.tokenname,
+          timestamp: decodedData.timestamp,
```

Comment:
we *encode* the timestamp so it generates a new token string for the given username and tokenname, but I don't think we should *return* the timestamp, just the username and tokenname